app/src/main/java/com/ayush/weatherapp/home/HomeContract.java

```diff
@@ -1,24 +1,22 @@
 package com.ayush.weatherapp.home;
 
-import android.support.annotation.DrawableRes;
 import com.ayush.weatherapp.constants.Temperature;
 import com.ayush.weatherapp.entities.CurrentForecastEntity;
 import com.ayush.weatherapp.entities.DailyDataEntity;
 import com.ayush.weatherapp.entities.HourlyDataEntity;
 import com.ayush.weatherapp.mvp.BaseContract;
-import com.ayush.weatherapp.retrofit.weatherApi.pojo.CurrentForecastDTO;
-import com.ayush.weatherapp.retrofit.weatherApi.pojo.DailyDataDTO;
-import com.ayush.weatherapp.retrofit.weatherApi.pojo.HourlyDataDTO;
 import java.util.List;
 
 public interface HomeContract {
   interface View extends BaseContract.BaseView {
     void setCurrentForecast(CurrentForecastEntity currentForecastDTO);
 
-    void setCurrentTemperature(String temperature);
+    void setCurrentTemperature(int temperature, @Temperature int tempUnit);
```

Comment:
We could make this dumb by changing the parameter to String and calculating in presenter